examples/batch/argo-workflows-batch/README.md

```diff
@@ -0,0 +1,239 @@
+## Batch processing with Argo Worfklows
+
+In this notebook we will dive into how you can run batch processing with Argo Workflows and Seldon Core.
+
+Dependencies:
+
+* Seldon core installed as per the docs with an ingress
+* Minio running in your cluster to use as local (s3) object storage
+* Argo Workfklows installed in cluster (and argo CLI for commands)
+
+
+## Setup
+
+### Install Seldon Core
+Use the notebook to [set-up Seldon Core with Ambassador or Istio Ingress](https://docs.seldon.io/projects/seldon-core/en/latest/examples/seldon_core_setup.html).
+
+Note: This example won't work in KIND (Kubernetes in Docker) due to a known requirement from Argo which mounts the docker socket: https://github.com/argoproj/argo/issues/826
+
```

Comment:
I managed to get this running with fix in https://github.com/argoproj/argo/issues/2376#issuecomment-595593237
Can you add thos instructions for Kind.

Reply:
Nice, thanks - ok I've added the comment